src/lib/popover/popover.test.ts

```diff
@@ -1428,6 +1450,7 @@ interface IPopoverFixtureConfig {
   animationType?: PopoverAnimationType;
   triggerType?: PopoverTriggerType;
   persistentHover?: boolean;
+  hoverDelay?: any;
```

Comment:
Use `number` here instead and for your NaN test just set `hoverDelay: 'Testing' as any` to force override the type. We want strong typing for everything else on this property.